batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/controller/CommonHttpServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.batch7crud.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.servlet.annotation.WebServlet'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)